### PR TITLE
groups content type builer attributes validation

### PR DESCRIPTION
**Pull request by alexandrebodin**
#### Description of what you did:

Adds validatin for a group attributes in the content-type-builder

#### My PR is a:

- [ ] 💥 Breaking change
- [ ] 🐛 Bug fix
- [ ] 💅 Enhancement
- [ ] 🚀 New feature

#### Main update on the:

- [ ] Admin
- [ ] Documentation
- [ ] Framework
- [ ] Plugin

#### Manual testing done on the following databases:

- [ ] Not applicable
- [ ] MongoDB
- [ ] MySQL
- [ ] Postgres
- [ ] SQLite

### CODE REVIEW

File: packages/strapi-plugin-content-type-builder/controllers/validation/common.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+  'richtext',
+  'json',
+  'enumeration',
+  'password',
+  'email',
+  'integer',
+  'float',
+  'decimal',
+  'date',
+  'boolean',
```
Comment (soupette) [inline packages/strapi-plugin-content-type-builder/controllers/validation/common.js]: I believe we are also supporting `biginteger`

File: packages/strapi-plugin-content-type-builder/controllers/validation/types.js [hunk 2]
```diff
@@ -0,0 +1,132 @@
...
+    }
+    case 'integer': {
+      return {
+        default: yup.number().integer(),
+        required: validators.required,
+        unique: validators.unique,
+        min: yup
+          .number()
+          .integer()
+          .positive(),
```
Comment (soupette) [inline packages/strapi-plugin-content-type-builder/controllers/validation/types.js]: Does this mean that we can't set the minimum value of integer to `-2` ?
Same for the maximum

File: packages/strapi-plugin-content-type-builder/controllers/validation/common.js (hunk 1 above)
Comment (alexandrebodin) [reply packages/strapi-plugin-content-type-builder/controllers/validation/common.js]: not in the frontend

File: packages/strapi-plugin-content-type-builder/controllers/validation/types.js (hunk 2 above)
Comment (alexandrebodin) [reply packages/strapi-plugin-content-type-builder/controllers/validation/types.js]: Indeed changed it

File: packages/strapi-plugin-content-type-builder/controllers/validation/common.js (hunk 1 above)
Comment (soupette) [reply packages/strapi-plugin-content-type-builder/controllers/validation/common.js]: You right! We don't support it in the `content-type-builder` but we do in the `content-manager`...

**Pull request merged**